test/tall/regression/0000/0083.unit

```diff
@@ -12,10 +12,14 @@ main() {
 main() {
   test('some string here', () {
     return aVeryLongFunctionThatNeedsToWrap(
-            toASecondLine, wrapWap, onThe, argsHere)
-        .then((_) {
+      toASecondLine,
+      wrapWap,
+      onThe,
+      argsHere,
+    ).then((_) {
       someLongFunctionHereToo(
-          'value value value value value value valvalue value value value');
+        'value value value value value value valvalue value value value',
+      );
```

Comment:
Here's another one where the old style was weird and busted but the new style looks good.